tools/bazar/presentation/templates/widget.tpl.html

```diff
@@ -108,11 +108,11 @@
             </div>
             <div class="form-group">
               <label>Latitude</label>
-              <input type="number" min="-90" max="90" v-model="latmodel" class="form-control">
+              <input type="text" min="-90" max="90" v-model="latmodel" class="form-control">
```

Comment:
will the attributes min and max work on type="text"?

Reply:
quand j'ia testé chez moi, le input était invalide et montré comme tel (entouré de rouge) parce que la valeur fournie (en js j'imagine) est une string avec des virgules

Mais oui si on passe en text on peut enlever min et max, tu valides?

Reply:
oui c'est un détail, juste je suis curieux pourquoi il invalide? il lui fallait un value="<un chiffre>" peut etre?

En tout cas c'est pas très important, fais comme ca t'arrange